tests/00_00_04-change-federation.js

```diff
@@ -0,0 +1,246 @@
+const rskUtils = require('../lib/rsk-utils');
+const { getRskTransactionHelpers } = require('../lib/rsk-tx-helper-provider');
+const { getBridge, getLatestActiveForkName } = require('../lib/precompiled-abi-forks-util');
+const { btcToWeis } = require('@rsksmart/btc-eth-unit-converter');
+const { removePrefix0x, ensure0x } = require('../lib/utils');
+const bridgeTxParser = require('bridge-transaction-parser-fingerroot500');
+const { expect } = require('chai');
+
+const { 
+    KEY_TYPE_BTC, 
+    KEY_TYPE_RSK, 
+    KEY_TYPE_MST,
+    REGTEST_FEDERATION_CHANGE_PRIVATE_KEYS,
+} = require('../lib/constants');
+
+// Generated with seed newFed1
+const newFederator1PublicKey = '0x02f80abfd3dac069887f974ac033cb62991a0ed55b9880faf8b8cbd713b75d649e';
+// Generated with seed newFed2
+const newFederator2PublicKey = '0x03488898918c76758e52842c700060adbbbd0a38aa836838fd7215147b924ef7dc';
+
+const expectedPendingFederationHash = '0x6f45ac8763317802a9a9c540b58af03e577bb4af14209f8a690d501e21c68ace';
+
+const expectedNewFederationAddress = '2NGJ9Rhk5KqK1RwMZm7Uph6nhGFL5MYtpJo';
+
+const expectedNewFederationErpRedeemScript = '0x64532102cd53fc53a07f211641a677d250f6de99caf620e8e77071e811a28b3bcddf0be12102f80abfd3dac069887f974ac033cb62991a0ed55b9880faf8b8cbd713b75d649e2103488898918c76758e52842c700060adbbbd0a38aa836838fd7215147b924ef7dc210362634ab57dae9cb373a5d536e66a8c4f67468bbcfb063809bab643072d78a1242103c5946b3fbae03a654237da863c9ed534e0878657175b132b8ca630f245df04db55ae6702f401b2755321029cecea902067992d52c38b28bf0bb2345bda9b21eca76b16a17c477a64e433012103284178e5fbcc63c54c3b38e3ef88adf2da6c526313650041b0ef955763634ebd2103776b1fd8f86da3c1db3d69699e8250a15877d286734ea9a6da8e9d8ad25d16c12103ab0e2cd7ed158687fc13b88019990860cdb72b1f5777b58513312550ea1584bc2103b9fc46657cf72a1afa007ecf431de1cd27ff5cc8829fa625b66ca47b967e6b2455ae68';
```

Comment:
Can't we get these values using powpeg-redeem-script-parser? So it's not hardcoded here, since this basically depends on the keys used in the config file. A change in the config would make this test fail

Reply:
Updated.